modules/ossm-about-collecting-ossm-data.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_support/ossm-collecting-ossm-data.adoc
+
+[id="ossm-about-collecting-ossm-data_{context}"]
+= About collecting service mesh data
+
+You can use the `oc adm must-gather` CLI command to collect information about your cluster, including features and objects associated with {ProductName}.
+
+To collect {ProductName} data with `must-gather`, you must specify the {ProductName} image: 
+
+`oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7:1.1.0`.
```

Comment:
I think it's worth calling out in either text or a note that (like the oc tool) the version of the must-gather tool you use must match your current OCP version.

We might want to replace the version number in the command with a version variable?

Reply:
"Install the version of the OpenShift Container Platform command line utility (the oc client tool) that matches your OpenShift Container Platform version."